modules/gapi/include/opencv2/gapi/garg.hpp

```diff
@@ -99,6 +99,29 @@ using GRunArg  = util::variant<
     >;
 using GRunArgs = std::vector<GRunArg>;
 
+// TODO: Think about the addition operator
+/**
+ * @brief This operator allows to complement input vector at runtime.
+ *
+ * It's an ordinary overload of addition assignment operator.
+ *
+ * Example of usage:
+ *
+ * @code
+ * cv::Mat inMat;
+ * GRunArgs inVec;
```

Comment:
Also I'd avoid referring to this type `cv::GRunArgs` as is, probably better to have `auto` here.

E.g.
```c++
auto inputs = cv:;gin();
if (something) {
    inputs += cv::gin(extraInMat);
}
```

Reply:
Done